src/components/UsersPhotoList/UsersPhotoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { PathList } from './photoPathList'
+
+type UsersPhotoListProps = {
+  styles?: string
+}
+
+const UsersPhotoList = (props: UsersPhotoListProps) => {
+  const { styles = '' } = props
+  return (
+    <div
+      className={`flex w-max items-center justify-center rounded-full p-2.5 ${styles}`}
+    >
+      {PathList.map((item, index) => (
```

Comment:
Don't use the index as a key.
Please read the cons of using the index as a key.